cluster/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -4,8 +4,8 @@ apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: rook-ceph-system
   labels:
-    operator: rook
-    storage-backend: ceph
+    app.kubernetes.io/operator: rook
+    app.kubernetes.io/storage-backend: ceph
```

Comment:
I think the storage-backend label is a rook-specific label. Maybe this should be `rook.io/storage-backend: ceph`?

Reply:
>What is use case behind this feature:
This allows you to support tooling which makes use of the recommended labels, the recommended labels describe applications in a way that can be queried, for example a kubectl can run on all csi resources in the rook-ceph namespace.
 
 The main purpose of `recommended labels` is just providing more information and easily querying?